resources/faq.md

```diff
@@ -0,0 +1,163 @@
+
+MeiliSearch is really **easy to use** and is accessible for all kinds of developers!
+
+[A quick tour](/guides/introduction/quick_start_guide.md) is available in this documentation to get the basics of MeiliSearch.
+
+We also provide a lot of tools, like [our SDKs](/resources/sdks.md), to help you easily integrate MeiliSearch in your project. New tools arrive every day!
+
+Plus, you can [contact us](/resources/faq.md#how-can-i-contact-the-meilisearch-team) if you need any help. We will answer for sure!
+
+## Do I need to configure MeiliSearch to get it working?
+
+MeiliSearch configuration works out-of-the-box. It means, by default, MeiliSearch handles all the configurations needed to provide a powerful and relevant search.
+
+For example, without any configuration, MeiliSearch is typo tolerant.<br>
+Type `craete an inedx` in the search bar of this documentation to experience the typo tolerance of our search engine.
+
+If you want to understand the relevancy of MeiliSearch, here is detailed and well-explained [information about the subject](/guides/main_concepts/relevancy.md#ranking-rules).
+
+Of course, MeiliSearch is still [highly customizable](/references/settings.md) to adapt the search to your needs with synonyms, stop words, and custom ranking rules.
+
+## How to know if MeiliSearch perfectly fits my use cases?
+
+MeiliSearch is open-source and really easy to use. It means that you can easily do a quick test with your data following our [getting started](/guides/introduction/quick_start_guide.md).
+
+Plus, we have already provided a [comparison with MeiliSearch alternatives](/resources/comparison_to_alternatives.md) for you.
+
+## Which languages can MeiliSearch handle?
+
+MeiliSearch works perfectly with English, kanji, and Latin based languages.<br>
+If you have any hesitation about your language handling, please contact us.
+
+## Do you provide a real dataset to test MeiliSearch?
+
+Here is a [movies dataset](https://www.notion.so/meilisearch/A-movies-dataset-to-test-Meili-1cbf7c9cfa4247249c40edfa22d7ca87#b5ae399b81834705ba5420ac70358a65) we provide to test MeiliSearch. More datasets are coming soon!
+
+## I did a call to an API route, and I only got an `updateId` as a response. What does it mean?
+
+MeiliSearch is an **asynchronous API**.<br>
+It means that, in a lot of cases (e.g., documents addition), you will receive as server response a simple JSON with only an `updateId` attribute. For example:
+
+```json
+{
+    "updateId": 2
+}
+```
+
+This kind of **successful response** indicates that the operation has been taken into account, but it may not have been executed yet.<br>
+
+::: tip
+You can check the status of the operation via this `updateId` and the [get update status route](/references/updates.md#get-an-update-status).<br>
+Also, MeiliSearch provides a global route to [track all your update status](/references/updates.md#get-all-update-status).<br>
+**Thus, you will know if your action succeeded or not, and <u>why</u>**.
```

Comment:
```suggestion
**This way, you will be informed if your action was successful or not, and <u>why</u>**.
```

Reply:
Finally, instead of `succeeded` or `was successful` I would rather use the `processed` term because that's the word that the MeiliSearch API returns in case of success :) Can you change you commit suggestion? (because I agree with the other changes in this sentence)

Reply:
You're right since it is asynchronous. Thank you, I updated it.